src/zope/i18nmessageid/_zope_i18nmessageid_message.c

```diff
@@ -141,6 +180,9 @@ Message_traverse(Message *self, visitproc visit, void *arg)
   Py_VISIT(self->domain);
   Py_VISIT(self->default_);
   Py_VISIT(self->mapping);
+  Py_VISIT(self->value_plural);
+  Py_VISIT(self->default_plural);
+  Py_VISIT(self->number);
```

Comment:
(TBH I'm surprised we need to visit immutables like strings and ints during garbage collection.)